lib/js/components/Form/Toggle/Toggle.vue

```diff
@@ -0,0 +1,305 @@
+<template>
+	<div
+		class="toggle"
+		:class="[
+			`-${size}`,
+			{
+				'-selected': isSelected,
+				'-default': state === TOGGLE_STATE.DEFAULT,
+				'-disabled': state === TOGGLE_STATE.DISABLED,
+				'-loading': state === TOGGLE_STATE.LOADING,
+			},
+		]"
+		@click="onToggle"
+	>
+		<div class="toggle__iconWrapper">
+			<icon :icon="icon" :size="iconSize" class="toggle__icon" />
+		</div>
+		<div v-if="label" class="toggle__label">{{ label }}</div>
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../../../styles/settings/animations';
+@import '../../../../styles/settings/colors/tokens';
+@import '../../../../styles/settings/spacings';
+@import '../../../../styles/settings/typography/tokens';
+
+$toggle-sizes: (
+	'x-small': (
+		'iconWrapperMargin': $space-xxxxs,
+		'iconBorder': $space-xxxs,
+	),
+	'small': (
+		'iconWrapperMargin': $space-xxs,
+		'iconBorder': $space-xxxs,
+	),
+	'medium': (
+		'iconWrapperMargin': $space-xxs,
+		'iconBorder': $space-xxs,
+	),
+);
+
+$toggle-colors: (
+	'default': (
+		'not-selected': (
+			'label': $color-neutral-text-heavy,
+			'icon': $color-neutral-icon,
+		),
+		'selected': (
+			'label': $color-neutral-text-heavy,
+			'icon': $color-primary-icon,
+		),
+		'hoverable': true,
+	),
+	'disabled': (
+		'not-selected': (
+			'label': $color-neutral-text-heavy-disabled,
+			'icon': $color-neutral-icon-disabled,
+		),
+		'selected': (
+			'label': $color-neutral-text-heavy-disabled,
+			'icon': $color-primary-icon-disabled,
+		),
+	),
+	'loading': (
+		'not-selected': (
+			'label': $color-neutral-text-heavy-disabled,
```

Comment:
loading should like `default`, not `disabled`: https://miro.com/app/board/o9J_knPnexA=/?moveToWidget=3458764546392448081&cot=14